torch/_export/passes/add_runtime_assertions_for_constraints_pass.py

```diff
@@ -112,28 +113,36 @@ def _insert_specialized_shapes_assert(self, arg, dims, name, current_inp):
                 f"specialized at {shapes[dim]}"
             )
             dim_node = super().call_operator(
-                torch.ops.aten.sym_size,
+                torch.ops.aten.sym_size.int,
                 (arg, dim),
                 {},
-                NodeMetadata({}),
+                NodeMetadata({
```

Comment:
Looks like this is repeated everywhere, maybe make a util? You can probably use a different function if you want to cut off the util call from the stack trace.

Reply:
Yep can do.